.expeditor/release.omnibus.yml

```diff
@@ -12,8 +12,8 @@ builder-to-testers-map:
   debian-8-x86_64:
     - debian-8-x86_64
     - debian-9-x86_64
-  el-6-i386:
-    - el-6-i386
+  el-6-i686:
+    - el-6-i686
```

Comment:
Um. Is this a dramatic change in architecture we support or only a labeling change?

Reply:
only a labeling change as mentioned in the PR comment. good catch though.

Reply:
I saw "Fix platform names to conform with what the new package publishing code expects" in the commit message, but didn't interpret that as assurance that architecture wasn't actually changing. Thanks!